test/lib/collectionFactory.test.js

```diff
@@ -17,7 +17,8 @@ describe('Collection Factory', () => {
         userId,
         name,
         description,
-        pipelineIds
+        pipelineIds,
+        type: undefined
```

Comment:
I don't think `undefined` is allowed
can you switch this to something else?
https://github.com/screwdriver-cd/data-schema/blob/master/models/collection.js#L49